benchmarks/run.py

```diff
@@ -182,7 +169,10 @@ def main():
             test = test_class(True)
             for j in range(3):
                 for case in cases:
-                    passed, result = test.validate(case)
+                    if json:
```

Comment:
i think we should try and omit the parsing/validation time from the json test.

Can we do something like

```py
if json:
    models = [m for passed, m in (test.validate(c) for c in cases) if passed]
    start = datetime.now()
    ...
        ...
        for m in models:
            test.to_json(m)
```

Reply:
Certainly, I did want to exclude the parsing time as well. I wasn't sure how deep we'd want to get with that bit.